low-level/sensors/impl/generic/node_hw.py

```diff
@@ -1277,60 +1273,57 @@ def _parse_voltage_info(self, index, date, _time, sensor, sensor_num, event, sta
         self._send_json_msg(resource_type, alert_type, severity, info, specific_info)
         store.put(self.faulty_resources, self.faulty_resources_path)
 
-# TODO: Enable this code once Intel servers become available
-# to test the current sensor
-#    def _parse_current_info(self, index, date, _time, sensor, sensor_num, event, status, is_last):
-#
-#        sensor_name = self.sensor_id_map[self.TYPE_CURRENT][sensor_num]
-#        resource_type = NodeDataMsgHandler.IPMI_RESOURCE_TYPE_CURRENT
-#
-#        threshold = event.split(" ")[-1]
-#        if threshold.lower() in ['low', 'high']:
-#            alert_type = f"threshold_breached:{threshold}"
-#        if alert_type:
-#            severity_reader = SeverityReader()
-#            severity = severity_reader.map_severity(alert_type)
-#            if threshold.lower() in ['low', 'high'] and status.lower.lower() == "deasserted":
-#                severity = "informational"
-#        else:
-#            alert_type = "miscellaneous"
-#            severity = "informational"
-#
-#
-#        common, specific, specific_dynamic = self._get_sensor_props(sensor_name)
-#
-#        specific_info = {}
-#        specific_info.update(common)
-#        specific_info.update(specific)
-#        specific_info.update({'fru_id': sensor})
-#        if is_last:
-#            specific_info.update(specific_dynamic)
-#
-#        info = {
-#            "site_id": self._site_id,
-#            "rack_id": self._rack_id,
-#            "node_id": self._node_id,
-#            "cluster_id":self._cluster_id ,
-#            "resource_type": resource_type,
-#            "resource_id": sensor_name,
-#            "event_time": self._get_epoch_time_from_date_and_time(date, _time),
-#            "description": event
-#        }
-#
-#        if (threshold.lower() in ['low', 'high'] and status.lower() == "asserted"):
-#            self.faulty_resources[sensor_name] = {
-#                'status': status,
-#                'event': event,
-#                'device_id': sensor,
-#                'fru_type': self.TYPE_VOLTAGE
-#            }
-#        elif (alert_type in ['fault_resolved', 'miscellaneous'] or \
-#            (threshold.lower() in ['low', 'high'] and status.lower() == "deasserted")) \
-#                and sensor_name in self.faulty_resources:
-#            del self.faulty_resources[sensor_name]
-#
-#        self._send_json_msg(resource_type, alert_type, severity, info, specific_info)
-#        store.put(self.faulty_resources, self.faulty_resources_path)
+    def _parse_current_info(self, index, date, _time, sensor, sensor_num, event, status, is_last):
+
```

Comment:
Please add docstring for function `_parse_current_info`

Reply:
done.